tests/serverpod_test_client/lib/src/custom_classes.dart

```diff
@@ -31,20 +31,80 @@ class CustomClass2 {
   dynamic toJson() => {'text': value};
 }
 
-class CustomClass3 implements SerializableModel, ProtocolSerialization {
-  final String value;
+class CustomClassWithoutProtocolSerialization {
```

Comment:
Suggestion: Make these a straight copy paste from the Server implementation.

This is most likely the scenario when our users are using custom classes.